docs/serviceability.md

```diff
@@ -0,0 +1,58 @@
+# Serviceability
+
+## Logs
+
+### Deployed Locally
+View the application-service controller logs by running the manager executable. Example,
+
+```
+make install
+make build
+./bin/manager
+```
+
+### Deployed on a Cluster
+View the application-service controller logs by tailing the manager container log of the controller manager pod. Example,
+
+```
+oc logs -f application-service-application-service-controller-manager -c manager
+```
+
+### Understanding the Logs
+Each application-service controller logs their reconcile logic to the manager. The log message format is generally of format 
+
+```
+2023-03-14T08:39:25.009Z	INFO	controllers.Component	updating devfile component name kubernetes-deploy ...	{"appstudio-component": "HAS", "Component": "mynamespace/pipeline-devfile-example-lcec", "clusterName": ""}
+```
+
+The above log indicates that the message is logged by the `Component` controller for the resource `pipeline-devfile-example-lcec` in the namespace `mynamespace`.
+
+## Debugging
+
+- Insert break points at the controller functions to debug unit tests
```

Comment:
This only explains debugging unit tests which is pretty narrow if there is a real problem.  For managed services, we would typically get a copy of the logs, scan it for errors and then search the code repo with the error message snippets to see where the problem may be coming from but we need to be careful since some messages may be concatenated so searching on a unique word may be better.

Reply:
I put in more info on how to debug from the code.